spec/controllers/control_panel/stories_controller_spec.rb

```diff
@@ -0,0 +1,252 @@
+require 'rails_helper'
+
+RSpec.describe ControlPanel::StoriesController do
+  context "user logged in" do
+    let(:user)  { create :user }
+    let(:board) { create :board, owner_id: user.id }
+    let(:story) { create :story, board_id: board.id}
+
+    context "User signed in" do
+      before { sign_in(user) }
+
+      describe " GET #index" do
+        it "renders stories#index" do
+          get :index, board_id: board.id
+          expect(response).to render_template :index
+        end
+      end
+
+      describe "GET #new" do
+        it "renders stories#new" do
+          get :new, board_id: board.id
+          expect(response).to render_template :new
+        end
+      end
+
+      describe "POST #create" do
+        let(:request) { post :create, board_id: board.id, story: { title: 'cokolwiek', estimate: 1} }
+
+        context "success" do
+          it "changes count of Stories" do
+            expect { request }.to change { Story.count }.by(1)
+          end
+
+          it "redirects properly" do
+            request
+            expect(response).to redirect_to control_panel_board_path(story.board_id)
+          end
+
+          it "gives 302 http code" do
+            request
+            expect(response).to have_http_status(302)
+          end
+
+          it "flashes message" do
+            request
+            expect(flash[:notice]).to eq I18n.t('shared.created')
+          end
+        end #context 'success'
+
+        context "failure" do
+          before do
+            allow_any_instance_of(Story).to receive(:save) { false }
```

Comment:
I'm not sure about that you should use `allow` only when you can't get false from save in simple way or if you don't want to create many object for one spec. In this case you can simply create story with invalid value and get an error (from validation).